src/main/kotlin/blackjack/model/FinalResult.kt

```diff
@@ -0,0 +1,24 @@
+package blackjack.model
+
+import blackjack.controller.Controller.Companion.BLACKJACK_SCORE
+
+class FinalResult(val dealer: Player, players: List<Player>) {
```

Comment:
How about wrapping `players` as well?

Reply:
I'd also suggest adding tests for this class, since there's logic in the constructor.